docs/contributing-to-piwik-plugins.md

```diff
@@ -0,0 +1,12 @@
+---
+category: DevelopInDepth
+---
+
+# Contributing to Matomo Plugins
+
+### Commit messages to not trigger any release.
```

Comment:
@AltamashShaikh might be a bit misleading headline as it suggests that a commit message might trigger a release?

Reply:
does this looks fine ?